examples/typescript/linkedin_ai_assistant/lib/browser_setup.ts

```diff
@@ -0,0 +1,37 @@
+import puppeteer, { Browser, Page } from 'puppeteer-core';
+
+let activeBrowser: Browser | null = null;
+let activePage: Page | null = null;
```

Comment:
@m13v fyi in typescript you usually use hyphen for file name and not lower case (like in rust, python)